db/schema.rb

```diff
@@ -10,11 +10,109 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20190613153704) do
+ActiveRecord::Schema.define(version: 20190626184757) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "courses", force: :cascade do |t|
+    t.string   "name"
+    t.integer  "teacher_id"
+    t.integer  "standards_chart_id"
+    t.datetime "created_at",         null: false
+    t.datetime "updated_at",         null: false
+    t.index ["standards_chart_id"], name: "index_courses_on_standards_chart_id", using: :btree
+    t.index ["teacher_id"], name: "index_courses_on_teacher_id", using: :btree
+  end
+
+  create_table "decks", force: :cascade do |t|
+    t.date     "release_date"
+    t.string   "name"
+    t.text     "description"
+    t.integer  "teacher_id"
+    t.datetime "created_at",   null: false
+    t.datetime "updated_at",   null: false
+    t.index ["teacher_id"], name: "index_decks_on_teacher_id", using: :btree
+  end
+
+  create_table "decks_questions", id: false, force: :cascade do |t|
+    t.integer "deck_id",     null: false
+    t.integer "question_id", null: false
+    t.index ["deck_id", "question_id"], name: "index_decks_questions_on_deck_id_and_question_id", using: :btree
+    t.index ["question_id", "deck_id"], name: "index_decks_questions_on_question_id_and_deck_id", using: :btree
+  end
+
+  create_table "periods", force: :cascade do |t|
```

Comment:
@xnanodax @xnanodax I wonder if we can instead call this `classes` and call `period_students` enrollments instead?

Reply:
Decided to leave this as periods. The meaning of periods seems less ambiguous than class. Although it might be an issue if teachers don't have periods? Not sure.